src/components/card/card.component.jsx

```diff
@@ -0,0 +1,43 @@
+import "./card.style.css";
+/**
+ *
+ * @param {string} title - The title of the Card
+ * @param {string} size - The Size of card expected value either small or big 
+ * @param {object} data - The data will be displayed in card it will be represented as key: value 
+ * if data has one prop it will display its value only
+ 
+ */
+function Card({ title, data, size }) {
```

Comment:
I think we need to give `size` a default value as we call size.toLowerCase() which will throw an error in case someone forgot to provide it.